.github/workflows/build-test-prValidation.yml

```diff
@@ -111,7 +108,32 @@ jobs:
           -Dakka.test.multi-node=false \
           -Dsbt.log.noformat=false \
           -Dakka.log.timestamps=true \
-          Test/compile
+          validatePullRequest
+
+  check-binary-compatibility:
+    name: Check / Binary Compatibility
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        # The versions of scala specified here are only used as `-Dakka.build.scalaVersion=...`
+        # which ignores the PATCH portion of the version id. These Scala versions don't need
+        # to be fully defined here then since Akka build handles which patch version will be used.
+        scalaVersion: ["2.12", "2.13"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8.0
+
+      - name: Report MiMa Binary Issues
+        run: sbt -jvm-opts .jvmopts-ci -Dakka.build.scalaVersion=${{ matrix.scalaVersion }} ";update ;mimaReportBinaryIssues ;Test/compile ;validateCompile"
```

Comment:
I'm not sure we need `Test/compile` here, but it was part of Travis CI. Also, maybe move `validateCompile` to the job running `validatePullRequest`?

Reply:
I agree, compiling everything should be covered by other steps

also, that `update` shouldn't be needed?